base/plumbing/DataStore.js

```diff
@@ -705,56 +702,26 @@ class Store {
 			cachePeriod = Math.max(cachePeriod, 5000);
 		}
 
-		return this.fetch2(path, fetchFn, cachePeriod);
-	} // ./fetch()
-
-
-	/**
-	 * Turn whatever object or promise into a PromiseValue. No-op if it's already a PV.
-	 * @param {*} thing
-	 * @returns {PromiseValue}
-	 */
-	wrapPV(thing) {
-		if (thing instanceof PromiseValue) return thing;
-		return new PromiseValue(thing);
-	}
-
-
-	/** Convenience for "is null/undefined" (ie allowing 0, false, '' etc) */
```

Comment:
@roscoemcinerney I suggest we keep this method, even if it is just `=== null` as the place to store doc notes on handling of null and undefined.